src/server/src/main/java/io/cassandrareaper/jmx/JmxConnectionFactory.java

```diff
@@ -100,7 +99,8 @@ public final JmxProxy connectAny(
           try {
             return connect(handler, host, connectionTimeout);
           } catch (ReaperException | RuntimeException e) {
-            LOG.info("Unreachable host", e);
+            LOG.info("Unreachable host: {}: {}", e.getMessage(), e.getCause().getMessage());
+            LOG.debug("Unreachable host: ", e);
```

Comment:
this change simply keeps the informational stacktrace to debug, keeping only an informative message at the info level.